presentations/en/2019-07-xx-Mandate.md

```diff
@@ -50,41 +131,51 @@ Federal public servants are entrusted to use and care for public resources respo
 
 For example, node in the self service is 5 years out of date. As a result, there are missing security patches and are unable to use modern tools for validating the quality of work due to out of date functionality
 
+---
+
 ### Integration over creation
 
 Do not compete with the private sector. Teams must prioritize integrating ESDC services into the existing IT ecosystem over developing, or improving existing, client facing applications competing for Canadians' attention. Create jobs, not software.
 
+--
+
 #### Repercussions
 
 - Cancel any UI updates or modernizations for Job Bank
 
+--
+
 #### Opportunities
 
 - Assign resources to creating new ways of communicating with Canadians to better deliver services
   - Allow Canadians to interact with Job Bank data through Alexa, Google Home, or any other home automation product
   - Interact with Canadians about job opportunities through automated notifications through LinkedIn
   - Dedicate more resources focused exclusively on developing service interfaces
 
->delivering programs that help Canadians move through life’s transitions, from school to work, from one job to another, *from unemployment to employment* and from the workforce to retirement
+Delivering programs that help Canadians move through life’s transitions, from school to work, from one job to another, *from unemployment to employment* and from the workforce to retirement
 
-...
+---
 
 ### Avoid Toil
 
 Toil is the kind of work tied to running a production service that tends to be manual, repetitive, automatable, tactical, devoid of enduring value, and that scales linearly as a service grows.
 
+--
+
 #### Repercussions
 
 - Re-define success
   - https://twitter.com/STP_PTS/status/1144268119025885186
```

Comment:
euhm... should we add this on a public forum?
perhaps keep this in our backpocket only.
Instead, success should be :
re-define success: when ESDC's lines of businesses succeed, IT succeeds.

Reply:
Or add the quote to the presentation. It was there, but we can take it out.  The whole last section needs more review.